test/models/common/checkpoints.py

```diff
@@ -81,13 +84,17 @@ def validate_checkpoint(
     ), "Model outputs should initially be different"
 
     # Save checkpoint from model 1 and load it into model 2
+    print("\n\nTest the save, load function ")
```

Comment:
Can we remove the added print statements?

Reply:
fixed